src/transport.js

```diff
@@ -114,6 +114,7 @@ if (!Util.isBrowser()) {
         method: options.method,
         headers: headers,
         body: options.data,
+        ...(options.signal && {signal : options.signal})
```

Comment:
Why not write "signal: options.signal" ?
The SDK does not currently use any build tools, and hence trying to keep the syntax as simple as possible.

Reply:
Also, what about the axis transport layer? Both fetch and axis transport layers should be updated